web-app/js/portal/filter/BooleanFilterPanel.js

```diff
@@ -41,8 +41,7 @@ Portal.filter.BooleanFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
     },
 
     _formatBoxLabel: function() {
-        var label = this.filter.label.split('_').join(' ').toTitleCase();
-        return label;
+        return this.filter.getDisplayLabel();
```

Comment:
I'm actually thinking that there will be somewhere else that has a function that takes a Filter and turns it into CQL. I've been thinking about Filters as a data structure rather than as an object with behaviour/business logic.